meta-resin-common/recipes-connectivity/openvpn/openvpn_%.bbappend

```diff
@@ -13,12 +12,13 @@ SRC_URI_append = " ${@bb.utils.contains("RESIN_CONNECTABLE","1","${RESIN_CONNECT
 
 RDEPENDS_${PN} += "${@bb.utils.contains("RESIN_CONNECTABLE","1","bash jq resin-device-uuid","",d)}"
 
+RDEPENDS_${PN}_append = " coreutils"
```

Comment:
This is still a resin specific configuration so it needs to go in the above line.